docs/content/entity-templates.md

```diff
@@ -53,6 +53,60 @@ public static class CreatureTemplate
 
 You can use this entity template to spawn a Creature entity. This can be done using the [MonoBehaviour](gameobject/world-commands.md) or [ECS](ecs/world-commands.md) workflow.
 
+### Feature Module components
```

Comment:
another question to this doc, unrelated to the actual changes (sorry), but we never specify what kind of entity we mean here. SpatialOS entity or Unity ECS entity, @ElleEss  what would be the best way to do this?

Reply:
Yeah, my first pass had a comment at the top that the existing text on here needs work - it's not so clear - loads of passive voice, not clear what type of entity it is....the first half needs a review....

Reply:
The disadvantage of reviewing docs in PRs is that you can't see them in context or make suggestions about unedited content which might need updating. Docs exist very much in context so the human-brain compiler can make sure it all makes sense.

Reply:
creating new ticket for Entity Template doc review (UTY-1252) + moving discussion to gdoc 

going to merge these changes in to unblock